make-sync-function

```diff
@@ -20,7 +20,7 @@ var outputFilename = process.argv[3];
 
 var syncFunc;
 try {
-  syncFunc = syncFunctionLoader.load(syncDocDefnFilename, scriptDir);
+  syncFunc = syncFunctionLoader.load(syncDocDefnFilename);
```

Comment:
Minor: now that `scriptDir` is no longer referenced, its definition can be removed from this file.